src/mumble/Settings.h

```diff
@@ -183,7 +184,7 @@ struct Settings {
 	int iQuality, iMinLoudness, iVoiceHold, iJitterBufferSize;
 	int iNoiseSuppress;
 	bool bAllowLowDelay;
-	bool bDenoise;
+	bool bDenoise; //TODO: replace with NoiseCancel noiseCancel;
```

Comment:
I just wanted to request this as a change. That'll also require a little change in the UI code though. If you want, I could help on that end :)